h2/src/main/org/h2/schema/Schema.java

```diff
@@ -63,9 +64,10 @@
      * @param owner the owner of the schema
      * @param system if this is a system schema (such a schema can not be
      *            dropped)
+     * @param tableEngineParams Table engine params to use when none are present in CREATE TABLE.
      */
     public Schema(Database database, int id, String schemaName, User owner,
-            boolean system) {
+            boolean system, ArrayList<String> tableEngineParams) {
```

Comment:
I don't think it makes sense to add this parameter to the constructor if we have a setter method.